packages/map/src/CdcMap.jsx

```diff
@@ -1277,6 +1279,13 @@ const CdcMap = ({ className, config, navigationHandler: customNavigationHandler,
     }
   }, [state.dataTable])
 
+    // When geo label override changes
```

Comment:
nit: subjective, but I'd recommend keeping comments together above the useEffect instead of breaking them between lines.

Reply:
OK changed it but I agree it really is a nit :-)